docs/source/gbd2017_models/concept_models/vivarium_gates_bep/concept_model.rst

```diff
@@ -0,0 +1,353 @@
+
+Iron and folic acid (IFA): Worldwide prevalence of iron deficiency anaemia in pregnancy is about 20%. Maternal anemia due to iron deficiency is associated with increased risk of maternal mortality, perinatal mortality and infants with low birth weight (LBW). Folic acid deficiency can lead to haematological consequences and congenital malformations; however, association with other birth outcomes is equivocal. WHO recommends iron and folic acid (IFA) supplementation for women during the antenatal period as part of routine antenatal care (ANC). The recommended dose of iron ranges from 30 mg to 60 mg. Despite its provision as part of national antenatal care programs for the last few decades in most low- and middle-income countries, coverage with the supplement is low. 
+
+Multi-micronutrients (MMN): Vitamin A deficiency is measured in night blindness and serum retinol levels. Globally, around 15% of pregnant women are deficient in vitamin A. Supplementation during pregnancy has been shown to reduce risk of maternal anaemia, infection and night blindness, but not on birth outcomes. Zinc deficiency has been associated with complications of pregnancy and delivery such as pre-eclampsia, premature rupture of membranes and congenital abnormalities. However, a review of trials of zinc supplementation showed a reduction in pre-term births only. (other micro-nutrients not reviewed here as they are not GBD risk factors). There is some evidence of additional benefit of MMN supplements containing 13–15 different micronutrients (including iron and folic acid) over iron and folic acid supplements alone, but there is also some evidence of risk, and some important gaps in the evidence. As such, WHO does not currently recommend a multi-nutrient supplement for pregnant women to improve maternal and perinatal outcomes. 
+
+Balanced energy protein (BEP): Under-nourished women are negatively associated with fetal growth and have increased risk of preterm birth. Balanced protein (where protein content provides less than 25% of total energy content) supplementation during pregnancy may help with gestational weight gain and subsequently avert adverse pregnancy outcomes, but the benefits are unclear. WHO recommends BEP for populations or settings with a high prevalence of undernourished pregnant women (not for individual pregnant women identified as being undernourished) to reduce the risk of stillbirths and small for-gestational-age neonates. Undernourishment is usually defined by a low BMI (i.e. being underweight BMI<18.5). For adults, a 20–39% prevalence of underweight women is considered a high prevalence of underweight and 40% or higher is considered a very high prevalence. MUAC may also be useful to identify protein–energy malnutrition in individual pregnant women and to determine its prevalence in this population. However, the optimal cut-off points may need to be determined for individual countries based on context-specific cost–benefit analyses. 
```

Comment:
I has been unclear to me whether MMN *inherently* contains IFA and whether BEP *inherently* contains IFA and MMN (as part of their ingredients) or whether it is *advised* that IFA is taken with MMN and that IFA and MMN are taken with BEP. Maybe this would be good to clarify here.

Reply:
you are right! it is confusing. I've clarified in the new concept model update. Namely, I am referring to MMN as oMMN to mean micro-nutrients _other_ than folic acid and iron. Additionally, I have grouped things into 1) effect sizes (that of IFA, that of oMMN, and that of BEP) 2) interventions and 3) scenarios. The interventions are an additive combination of effect sizes and the scenarios are a combination of interventions.